### DISCUSSION TITLE
Spread operator removes virutal properties

### DISCUSSION THREAD

**Discussion by curiouslad** [mikro-orm/mikro-orm#6211] [General]
### Describe the bug

after using spread operator on entity instance, virutal properies are removed. 
using spread with ...wrap(instance).toJSON() works (does not remove virtual poperties)

### Reproduction

**Entity({ tableName: 'Event', repository: () => EventsRepository })
export class Event extends BaseEntity {

  @Property({ persist: false })
  get coverImages() {
    return this.images.getItems().filter((i) => i.type === EventImageType.COVER);
  }

  @Property({ persist: false })
  get sponsorLogos() {
    return this.images.getItems().filter((i) => i.type === EventImageType.SPONSOR_LOGO);
  }

  @OneToMany(() => EventImage, (eventImage) => eventImage.event)
  images = new Collection<EventImage>(this);
}**
console.log({...event})

### What driver are you using?

@mikro-orm/postgresql

### MikroORM version

6.3.13

### Node.js version

20

### Operating system

ubuntu

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Discord](https://discord.gg/w8bjxFHS7X).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
First of all, I would strongly discourage spreading entities in the first place, why would you want to do that? This is not about virtual properties, its about getters in general.

```ts
class Foo {
    get foo() {
        return 'bar';
    }
}

const f = new Foo();
console.log(f);
console.log({ ...f });
```
